ksqldb-engine/src/main/java/io/confluent/ksql/function/UdtfLoader.java

```diff
@@ -67,7 +68,8 @@ public void loadUdtfFromClass(
     }
     final String functionName = udtfDescriptionAnnotation.name();
     final String sensorName = "ksql-udtf-" + functionName;
-    FunctionLoaderUtils.addSensor(sensorName, functionName, metrics);
+
+    FunctionMetrics.initInvocationSensor(metrics, sensorName, "ksql-udtf", functionName + " udtf");
```

Comment:
if anyone was relying on the old metrics, their alerts for udtfs will break with this change. probably ok as I'd be shocked if anyone was actually writing their own UDTFs _and_ using metrics on them at this point but something to call out 😉

Reply:
Yeah, I'm aware, but ... meh.  turning metrics on caused an NPE, so likely no one is using them, and better to change it now.

Reply:
As mentioned in the description, I see this as a bug fix.